### PR TITLE
Update the maintainers list

### PR DESCRIPTION
**Pull request by kashifest**
as the title says

### CODE REVIEW

**Comment by kashifest** [comment]
cc @dtantsur @lentzi90 @tuminoid @Rozzii

File: maintainers/ALL-OWNERS [hunk 1]
```diff
@@ -13,6 +13,7 @@ approvers:
 - kashifest
 - lentzi90
 - mboukhalfa
+- smoshiur1237
```
Comment (tuminoid) [inline maintainers/ALL-OWNERS:16]: This list is incorrect, see #367 for script fix to generate correct list.

Comment (tuminoid) [reply maintainers/ALL-OWNERS:19]: Now its better.

**Review by Rozzii — APPROVED** [review]
/approve

**Pull request merged**